skimage/measure/_regionprops_utils.py

```diff
@@ -326,3 +327,28 @@ def perimeter_crofton(image, directions=4):
 
     total_perimeter = coefs @ h
     return total_perimeter
+
+def check_spacing(spacing, ndims:int):
```

Comment:
This function transforms input so that it meets a norm, so I think the "check" is a bit misleading (it implies to me that the input is just checked and not changed). Also as long as we don't make this function part of the public API we should prefix it with `_`.
 
```suggestion
def _normalize_spacing(spacing, ndims):
```